Dockerfile

```diff
@@ -18,6 +18,11 @@ WORKDIR /ethconnect
 COPY --from=builder /ethconnect/ethconnect .
 COPY --from=builder /ethconnect/ethbinding.so .
 COPY --from=builder /ethconnect/start.sh .
-RUN ln -s /ethconnect/ethconnect /usr/bin/ethconnect
+
+# get the latest CA certs and symlink the ethconnect binary
+RUN apt-get update \
+    && apt-get install -y ca-certificates \
```

Comment:
Do we need to run `update-certificates` too?
https://github.com/hyperledger/firefly-ethconnect/blob/40e4ef0ef8f8b0db936c5c99f0d9df1bdf9eb2fa/Dockerfile#L23

Reply:
And maybe `rm -rf /var/lib/apt/lists/*` at the end of the `RUN` block to trim the docker image size down?